kombu/transport/SQS.py

```diff
@@ -402,18 +402,23 @@ def _put(self, queue, message, **kwargs):
         else:
             body = dumps(message)
         kwargs = {'QueueUrl': q_url, 'MessageBody': body}
-        if queue.endswith('.fifo'):
-            if 'MessageGroupId' in message['properties']:
-                kwargs['MessageGroupId'] = \
-                    message['properties']['MessageGroupId']
-            else:
-                kwargs['MessageGroupId'] = 'default'
-            if 'MessageDeduplicationId' in message['properties']:
-                kwargs['MessageDeduplicationId'] = \
-                    message['properties']['MessageDeduplicationId']
-            else:
-                kwargs['MessageDeduplicationId'] = str(uuid.uuid4())
 
+        if 'properties' in message:
```

Comment:
We added this check because one of the tests checks whether a raw message not created by kombu (hence not having a `properties` value) will still get passed properly to SQS. 

```
        # In test_messages_to_python()
        # json formatted message NOT created by kombu
        for i in range(json_message_count):
            message = {'foo': 'bar'}
            self.channel._put(self.producer.routing_key, message)
```
ref: https://github.com/celery/kombu/blob/master/t/unit/transport/test_SQS.py#L359

Added the check to include the checks for the FIFO params, because a raw message like this sent to a fifo queue would have errored out as well when checking for `properties`.